.github/workflows/build.yml

```diff
@@ -117,6 +118,8 @@ jobs:
       - name: Build Windows binaries
         if: inputs.release && inputs.platform == 'windows'
         shell: msys2 {0}
+        env:
+          RUSTFLAGS: "-C target-feature=+crt-static"
```

Comment:
no needed since we have cargo cfg

Reply:
https://github.com/awslabs/llrt/actions/runs/11089019083

If remove this env, it doesn't work, although the number of dlls is less, it still contains VCRUNTIME140_1.dll
I don't know how to fix this, so I just added it to ci

```
 ldd ./llrt.exe
        ntdll.dll => /c/WINDOWS/SYSTEM32/ntdll.dll (0x7ff9a1900000)
        KERNEL32.DLL => /c/WINDOWS/System32/KERNEL32.DLL (0x7ff99f810000)
        KERNELBASE.dll => /c/WINDOWS/System32/KERNELBASE.dll (0x7ff99ed10000)
        apphelp.dll => /c/WINDOWS/SYSTEM32/apphelp.dll (0x7ff99b630000)
        bcryptprimitives.dll => /c/WINDOWS/System32/bcryptprimitives.dll (0x7ff99f4c0000)
        WS2_32.dll => /c/WINDOWS/System32/WS2_32.dll (0x7ff9a05c0000)
        RPCRT4.dll => /c/WINDOWS/System32/RPCRT4.dll (0x7ff99fe60000)
        ucrtbase.dll => /c/WINDOWS/System32/ucrtbase.dll (0x7ff99f2c0000)
        bcrypt.dll => /c/WINDOWS/SYSTEM32/bcrypt.dll (0x7ff99e590000)
        VCRUNTIME140.dll => /c/WINDOWS/SYSTEM32/VCRUNTIME140.dll (0x7ff993df0000)
        CRYPTBASE.dll => /c/WINDOWS/SYSTEM32/CRYPTBASE.dll (0x7ff99dd00000)
        VCRUNTIME140_1.dll => /c/WINDOWS/SYSTEM32/VCRUNTIME140_1.dll (0x7ff97c3d0000)
        MSVCP140.dll => /c/WINDOWS/SYSTEM32/MSVCP140.dll (0x7ff97cbf0000)
```

Reply:
Add RUSTFLAGS
https://github.com/awslabs/llrt/actions/runs/11089325578

Reply:
Thank you for the clarification! Taking a step back, I’m curious about the potential benefits of compiling for MSVC instead of GNU. From what I can see, the total score of the benchmark appears to be the same. One challenge with using MSVC is that it limits our ability to cross-compile for Windows from a Linux host. On the Linux side, we’re statically including the C runtime because we're targeting musl, which isn’t widely available on most distros. Additionally, musl is more lightweight and can offer better performance, especially when combined with a custom allocator like mimalloc or snmalloc. Could you test if snmalloc on GNU offers any benefits, otherwise I think we should stick to GNU :)

Reply:
snmalloc does not support mingw, the only reason to use msvc is to reduce the dll dependency, so that the executable program can run on some old or lite version of windows, the performance gain is very limited, tjs also has this problem when using msys environment and releases the dll and executable together. 
I also prefer gnu, maybe add msvc support when llrt needs to handle this problem

https://github.com/saghul/txiki.js/blob/9076cb366a9707990a8c615b3c8178219f29e86e/.github/workflows/release.yml#L37

Reply:
Agree. I'll close the PR for now and we can revisit once there is a need! However I want to thank you for your contributions!